docs/install/full-install.rst

```diff
@@ -0,0 +1,14 @@
+=================
+Full Installation
+=================
+
+For a full Flocker installation, including the Flocker plugin for Docker, the following manual instructions take you through everything you need to do:
```

Comment:
At this point I'd like to know why I'd do a full installation instead of a quick one - we can't assume that people will have come from "Installing Flocker" or "Quick Start Flocker Installer" (something I learned at WTD :smile: ).

I'd remove / change "manual" - it isn't clear to me what a "manual" instruction is.